state/state.go

```diff
@@ -474,6 +474,11 @@ func (st *State) ModelUUID() string {
 	return st.modelTag.Id()
 }
 
+// ModelTag returns tag for the model controlled by state instance.
+func (st *State) ModelTAG() names.ModelTag {
```

Comment:
All our other call sites for things that return tags is to use a name like `ModelTag`.

Reply:
Agree. And its reasonable to do it this way. However, adding the ModelTag() in state object created ambiguous situation in many other facades which is out of scope for this task to resolve. I have tried and pushed a commit as well (which still needs some more fixes)wanted to have discussion on the same. Lets have it today.